zubhub_frontend/zubhub/src/assets/js/styles/views/project_details/projectDetailsStyles.js

```diff
@@ -106,28 +106,22 @@ const styles = theme => ({
   videoWrapperStyle: {
     backgroundColor: 'black',
     marginBottom: '1em',
-    height: '100%',
+    maxWidth: '880px',
+    maxHeight: '509px',
     paddingBottom: '56.25%',
-    [theme.breakpoints.down('1080')]: {
-      height: 0,
-    },
     [theme.breakpoints.down('959')]: {
       paddingBottom: '56.25%',
     },
   },
   iframeStyle: {
-    borderRadius: 6,
     overflow: 'hidden',
     position: 'absolute',
-    borderStyle: 'none',
-    top: 0,
-    left: 0,
+    top: '50%',
+    left: '50%',
     width: '100%',
     height: '100%',
-    [theme.breakpoints.down('959')]: {
-      width: '100%',
-      height: '100%',
-    },
+    transform: 'translate(-50%, -50%)',
```

Comment:
This will essentially ensure image dimensions remain consistent, but we see black bars always (since no image will always have exact same width x height ratio). I suggest we explore "zoom and fill" technique rather than this. Currently, we have a "stretch and fill"

Reply:
If I understood correctly, we should zoom the image to fill the container rather than showing bars around it right? Changing objectFit to 'cover' achieves this perfectly, which works well with most of the images other than very long or wide ones. In that case, only some part of the image is seen. 

here is a long image with what we currently have (stretched):
<img width="897" alt="Screenshot 2023-11-12 at 13 22 31" src="https://github.com/unstructuredstudio/zubhub/assets/92817363/cbd4593c-e32a-44eb-84ea-2afba50907ec">

here is the zoomed version of the long image:
<img width="897" alt="Screenshot 2023-11-12 at 13 21 12" src="https://github.com/unstructuredstudio/zubhub/assets/92817363/1daafff2-ffa4-4ab9-a5b7-df043e37428a">

and here is my first implementation using objectFit: contain: 
<img width="783" alt="Screenshot 2023-11-12 at 13 45 11" src="https://github.com/unstructuredstudio/zubhub/assets/92817363/9aef98aa-55e7-4383-80ad-f3c9f41fc784">

the third option may be unnecessary for other images but it allows us to show the whole content to the user. What do you think of using this option without the black background? This way the user sees the whole image and there won't be unnecessary black bars for each image.

Or we can use 'cover' and 'contain' conditionally depending on the image ratio.

Reply:
@brrkrmn I think lets do the zoom version. So third version can be removed. We already allow users to click the iamge and see a full screen original aspect ratio version.

Reply:
thanks @tuxology, made the last changes to use the zoom version, this is ready now 👍🏻